utils/world.ts

```diff
@@ -23,6 +23,7 @@ export interface WorldState {
   width: number;
   height: number;
   stats: Map<string, Stats>;
+  startTime: string;
```

Comment:
`startTime` должен быть типа `Date`.

`Date` включает год, день, месяц, часы, минуты, секунды, миллисекунды, в компактном формате – одно число.

```suggestion
  startTime: Date;
```